training-intern/src/utils/validateMultiForm.js

```diff
@@ -0,0 +1,299 @@
+import { updateDate } from "@/utils/index.js";
+
+export function validateLastForm(stepData) {
```

Comment:
các cái validate của form này đang bị hardcode quá, nghĩa là xử lý đặc trưng cho từng form. Giả sử sau này thêm 1 trường khác vào form thì lại phải sang đây viết thêm validate cho trường đó => không linh động.
A viết thành các hàm validate chung chung như validate maxLength, validate required..... đc k? xong vào file component nào mà chứa toàn bộ input, minhf check điều kiện của nó nếu có max-length thì validate max length chẳng hạn